packs/rabbitmq/sensors/queues_sensor.py

```diff
@@ -0,0 +1,85 @@
+import json
+import pickle
+
+import pika
+from pika.credentials import PlainCredentials
+
+from st2reactor.sensor.base import PollingSensor
+
+DESERIALIZATION_FUNCTIONS = {
+    'json': json.loads,
+    'pickle': pickle.loads
+}
+
+
+class RabbitMQSensor(PollingSensor):
```

Comment:
Commenting on this PR as well. We probably need a more specific class name. I am sure we'll write more rabbit integrations. (I am looking at you @manasdk :P)